engine/src/cmd/collide2/opcodetypes.h

```diff
@@ -150,34 +121,10 @@ typedef int64_t int64;
 // <stddef.h> provided the types by checking if _INTPTR_T_DEFINED has been
 // #defined; newer versions of MSVC will provide them; older ones will not.  If
 // all else fails, then we fake up these types on our own.
+//
+// 2021-06-26 SGT: I believe these are standard types now in C++11 and later.
```

Comment:
VS2010 or VS2012 started including hte `stddef.h` which has long defined the standard types on Unix/Linux systems. MS was the only oddball compiler left not providing it. I don't recall if the bit-types were pulled into ISO C or ISO C++ or not; it may still be by convention and everyone finally got onboard with providing them.